README.md

```diff
@@ -2089,28 +2089,29 @@ have in a company.
 
 In summary, you have to determine or make educated guesses about several things:
 
-- **Equity value**: Determined by the value the company may have in the future, and the number
-  of shares you may own?
+- **Equity value**: This can be estimated by the value the company may have in the future, and
+  the number of shares you may own.
   - **Percentage ownership**: As we’ve mentioned, knowing how many shares of stock or stock
-    options you have is meaningless unless you know the number of outstanding shares.
-    What matters is the percentage ownership of the company the shares represent, including
-    the [details](#counting-shares) of how the total is counted.
-  - **Risk**: It is critical to understand [risk](#stages-of-a-startup) in the business and dilution
-    to ascertain the possible future value of equity for the company and its stage.
+    options you have is meaningless unless you know the number of the company’s outstanding
+    shares. What matters is the percentage ownership of the company the shares represent,
+    including the [details](#counting-shares) of how the total is counted.
+  - **Risk**: It is critical to understand [risk](#stages-of-a-startup) in the business and the
+    principles of dilution to ascertain the possible future value of equity.
     [This article](http://codingvc.com/valuing-employee-options/) from Leo Polovets provides some
     additional thoughts.
-- **Vesting**: When you will receive the equity, as well as whether you’re able to exercise
-  stock options (and pay the associated costs and taxes).
-  And whether you can do all this before your exercise window expires.
-- **Liquidity**: When you’ll be able to sell the shares, and if that is likely to be for a
-  profit at that time.
+- **Vesting**: Understand when you will [receive the equity](#vesting-and-cliffs), as well as
+  whether you’re able to exercise stock options (and pay the associated costs and taxes),
+  and whether you can do all this before your exercise window expires.
+- **Liquidity**: Determine when you will be able to [sell your shares](#sales-and-liquidity), and
+  estimate at what point selling will turn a profit.
```

Comment:
Changes meaning: "at what point" implies it *will* happen but timing only is uncertain. Not true, since you often can't sell for profit.

Revert everything starting with "estimate".